frontend/public/features.ts

```diff
@@ -85,27 +93,30 @@ const detectOpenShift = dispatch => coFetchJSON(openshiftPath)
       : handleError(err, FLAGS.OPENSHIFT, dispatch, detectOpenShift)
   );
 
-const detectCanListNS = dispatch => {
-  const req = {
-    spec: {
-      resourceAttributes: {
-        resource: 'namespaces',
-        verb: 'list',
-      },
-    },
-  };
-  k8sCreate(SelfSubjectAccessReviewModel, req)
-    .then(
-      res => setFlag(dispatch, FLAGS.CAN_LIST_NS, res.status.allowed),
-      err => handleError(err, FLAGS.CAN_LIST_NS, dispatch, detectCanListNS)
-    );
-};
+const detectSSARRestrictions = _.transform([
+  [ FLAGS.CAN_LIST_NS, 'detectCanListNS', 'namespaces'],
+  [ FLAGS.CAN_LIST_NODE, 'detectCanListNode', 'nodes'],
+  [ FLAGS.CAN_LIST_PVC, 'detectCanListPVC', 'persistentvolumeclaims'],
```

Comment:
This should be persistentvolumes, not persistentvolumeclaims

Reply:
Will update.